README.md

```diff
@@ -50,7 +52,7 @@ sending emails.
 
 For example, making an initial configuration:
 
-    sqlite3 attestation.db "INSERT INTO Configuration VALUES ('emailUsername', 'alert@attestation.app'), ('emailPassword', '<password>'), ('emailHost', 'mail.grapheneos.org'), ('emailPort', '465')"
+    sqlite3 attestation.db "INSERT INTO Configuration VALUES ('emailUsername', 'alert@attestation.app'), ('emailPassword', '<password>'), ('emailHost', 'mail.grapheneos.org'), ('emailPort', '465');"
```

Comment:
The semicolon isn't required when using `sqlite3` as a CLI command instead of a shell, similar to using the C API for preparing/running a single statement.

Reply:
Removed semicolon